edit.js

```diff
@@ -0,0 +1,32 @@
+document.getElementById('add-experience-btn').addEventListener('click', function() {
+    document.getElementById('experience-form').style.display = 'block';
+});
+
+document.getElementById('experience-form').addEventListener('submit', function(event) {
+    event.preventDefault();
+
+    const company = document.getElementById('company').value;
+    const startDate = document.getElementById('start-date').value;
+    const endDate = document.getElementById('end-date').value;
+    const description = document.getElementById('description').value;
+
+    if (company && startDate && endDate && description) {
+        const experienceEntry = document.createElement('div');
+        experienceEntry.classList.add('experience-entry');
+        experienceEntry.innerHTML = `
+            <h3>${company}</h3>
+            <p>${startDate} - ${endDate}</p>
+            <p>${description}</p>
+        `;
+        document.getElementById('experience-entries').appendChild(experienceEntry);
+
+        document.getElementById('company').value = '';
```

Comment:
Given that the elements are already stored in variables, why is it necessary to retrieve them through selectors again?

Reply:
clears the value of the input field .This ensures that the form is reset and ready for a new entry after the user has submitted their experience.

Reply:
My concern is related to fields selection